### PR TITLE
Toggle icons for display mode

### PR DESCRIPTION
**Pull request by DylanC**
Raised during the UI/UX review:

> For Doctor, Eye icon for toggling display mode may not be the most appropriate choice.

I have replaced the static eye icon with both a light-mode and dark-mode icon. These icons will toggle when the button is clicked.

When in Dark mode:
image

When in Light mode:
image

### CODE REVIEW

File: visualizer/main.js [hunk 1]
```diff
@@ -15,7 +15,7 @@ const drawUi = () => {
   document.body.classList.add('is-font-loaded')
 
   menu.on('toggle-theme', function () {
-    document.documentElement.classList.toggle('light-theme')
+    document.documentElement.classList.toggle('toggle-theme')
```
Comment (goto-bus-stop) [inline visualizer/main.js:18]: why this name change?

Comment (DylanC) [reply visualizer/main.js:18]: @goto-bus-stop - Makes more sense this way. The button is not always to enable the light theme. Sometimes its to enable the dark theme. Following this logic we are toggling the theme. 

Its also to fall in line with the "toggle-grid" button which isn't called "toggle-four-boxes" for example. :smile:

**Comment by DylanC** [comment]
@goto-bus-stop - I'll look into changing the icon CSS today. I knew something looked a bit "off" with it.

File: visualizer/main.js (hunk 1 above)
Comment (goto-bus-stop) [reply visualizer/main.js:18]: But this class is applied to the body to enable the light theme, right? :thinking:

Comment (DylanC) [reply visualizer/main.js]: @goto-bus-stop - Fair enough! Can't argue with that. The toggle-theme will be light-theme again. :smile:

**Pull request merged**